hugolib/author.go

```diff
@@ -13,23 +13,51 @@
 
 package hugolib
 
+import (
+	"fmt"
+	"regexp"
+	"sort"
+	"strings"
+
+	"github.com/spf13/cast"
+)
+
+var (
+	onlyNumbersRegExp = regexp.MustCompile("^[0-9]*$")
+)
+
 // AuthorList is a list of all authors and their metadata.
 type AuthorList map[string]Author
 
+// Authors returns a slice of authors sorted by ID
+func (al AuthorList) Authors() []Author {
```

Comment:
Sure, but if the default sorts first by Weight, then by Name or whatever. I asked you to remove the sort methods because 1) They are very verbose and not implemented the way sorting is done elsewhere (on several levels) and 2) They have no tests. I don't want to add code that I know I'm gonna have to rework.

So they can be added later, the most important part being a sensible sort for the Author() func.